docs/victoriametrics/vmauth_flags.md

```diff
@@ -0,0 +1,238 @@
+---
+build:
+  list: never
+  publishResources: false
+  render: never
+sitemap:
+  disable: true
+---
+<!-- The file has to be manually updated during feature work in PR, make docs-update-flags command could be used peridically to ensure the flags in sync. -->
+```shellhelp
+
+vmauth authenticates and authorizes incoming requests and proxies them to VictoriaMetrics.
```

Comment:
The file still contains tabs; should be updated with the latest version of make docs-update-flags